tests/test_app.py

```diff
@@ -140,6 +162,127 @@ def test_app_init_with_invalid_name(self, name):
         with pytest.raises(ValueError):
             firebase_admin.initialize_app(CREDENTIAL, name=name)
 
+
+    @pytest.mark.parametrize('bad_file_name', ['firebase_config_empty.json',
+                                               'firebase_config_invalid.json',
+                                               'no_such_file'])
+    def test_app_init_with_invalid_config_file(self, bad_file_name):
+        config_old = set_config_env(bad_file_name)
+        with pytest.raises(ValueError):
+            firebase_admin.initialize_app(CREDENTIAL)
+        revert_config_env(config_old)
+
+
+    OptionsTestCase = namedtuple('OptionsTestCase',
```

Comment:
This is sitting in the middle of a class definition. Please move to somewhere more appropriate.